src/common/exif.cc

```diff
@@ -914,11 +914,11 @@ static bool _exif_decode_exif_data(dt_image_t *img, Exiv2::ExifData &exifData)
     {
       const float focus_position = pos->toFloat();
 
-      if (FIND_EXIF_TAG("Exif.Photo.FocalLengthIn35mmFilm")) {
-          const float focal_length_35mm = pos->toFloat();
+      if (FIND_EXIF_TAG("Exif.Photo.FocalLengthIn35mmFilm") && focus_position) {
+        const float focal_length_35mm = pos->toFloat();
 
-          /* http://u88.n24.queensu.ca/exiftool/forum/index.php/topic,3688.msg29653.html#msg29653 */
-          img->exif_focus_distance = (pow(2, focus_position / 16 - 5) + 1) * focal_length_35mm / 1000;
+        /* http://u88.n24.queensu.ca/exiftool/forum/index.php/topic,3688.msg29653.html#msg29653 */
+        img->exif_focus_distance = (pow(2, focus_position / 16 - 5) + 1) * focal_length_35mm / 1000;
```

Comment:
Now missing a fall-through case to set img->exif_focus_distance to something if `focus_position` is 0.0? In fact, a fall-through case seems to be missing in general? (Unless one assumes the default value is set correctly when `img` is constructed...?)

Reply:
Yes, a fall-through case was missing in original code. And I would add it for readability and reliability of the code (it is always better not to rely on the fact that somewhere else everything is done correctly). But this would be a larger patch, it is not very appropriate before the deadline and it does not address this particular issue.

And yes, I checked in the debugger that default value for `img->exif_focus_distance` was set correctly.